mkdocs.yml

```diff
@@ -217,9 +247,15 @@ markdown_extensions:
   - pymdownx.mark
   - pymdownx.tilde
   - attr_list
+  - md_in_html
   - tables
   # - mdx_truly_sane_lists
 extra:
+  version:
+    provider: mike
+  # - make edgexversion the latest version for repository tree references
+  edgexversion: 'v3.0.0'
+  edgexversionmain: 'main'
```

Comment:
This is not needed. all references to `main` with be updated to `{{edgexversion}}` when we release.

```suggestion
```